Dockerfile

```diff
@@ -8,7 +8,6 @@ COPY . /go/src/github.com/seatgeek/docker-mirror/
 RUN dep ensure -vendor-only
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o build/docker-mirror  .
 
-FROM alpine:latest
```

Comment:
i don't think there's any issue copying (and using) the go binary built in the previous alpine layer on our main docker-base (ubuntu) layer

Reply:
this also is fine, just checked in locally built docker image. (error is because config.yaml isn't mounted into image)
```
root@56a97be74b79:/# docker-mirror
FATA[0000] Could not read config file: open config.yaml: no such file or directory
```